### PR TITLE
Amended buttons for categories and items

### PR DESCRIPTION
**Pull request by Exiled184**

### CODE REVIEW

File: main_app/templates/categories/detail.html
Comment (MartyEternal) [inline main_app/templates/categories/detail.html:1]: CSS clean up

File: main_app/templates/items/detail.html
Comment (MartyEternal) [inline main_app/templates/items/detail.html:1]: CSS clean up.

File: main_app/templates/items/detail.html [hunk 1]
```diff
@@ -41,15 +41,15 @@ <h3>{{ item.name }}'s Categories</h3>
       <div class="card-action">
         <form action="{% url 'unassoc_category' item.id category.id %}" method="POST">
           {% csrf_token %}
-          <button type="submit" class="btn">Remove</button>
+          <button type="submit" class="btn btn-danger">Remove</button>
```
Comment (MartyEternal) [inline main_app/templates/items/detail.html:44]: Button colour change?

**Review by MartyEternal — COMMENTED** [review]
All checked. I only see button changes and CSS changes here and there.

**Review by jackson-07 — APPROVED** [review]
LGTM!

**Pull request merged**